app/experimenter/experiments/models.py

```diff
@@ -1078,6 +1078,12 @@ def clone(self, name, user):
 
         return cloned
 
+    def prepare_rapid_experiment_for_publish(self):
+        self.status = Experiment.STATUS_ACCEPTED
+        self.proposed_start_date = datetime.date.today()
```

Comment:
Ah, this reminds me... I wanted to make sure I confirmed that we have oodles of date fields in the Experimenter side. Mainly just for date that the submitter hit Save/Submit button, and the date the experiment was approved in RemoteSettings/Experimenter, and the date the experiment was set to "Live" (which I assume is the same date as approved date above).

Or more accurately, I want to know if somebody can run some queries against production to tell us what the average/median/mean times are for an experiment to actually be approved and what our outliers are (ie: an experiment that sat there for 2 weeks, or average time is 4 days, etc). I know the average will most likely be over 24 hours due to weekends and timezones, but just want to know if we can determine averages and stats.

Reply:
Oh yeah the way we track those is through ChangeLogs.  You just reminded me I need to add one for this publish step, thanks!